packages/riipen-ui/src/components/ButtonIcon.jsx

```diff
@@ -22,6 +22,7 @@ class ButtonIcon extends React.Component {
      */
     color: PropTypes.oneOf([
       "default",
+      "muted",
```

Comment:
Just used the Text component wording, but maybe this should be "grey" or something else?

Reply:
i think this is fine